src/components/MessagesList/MessagesList.vue

```diff
@@ -643,8 +706,60 @@ export default {
 			}
 		},
 
-		handleMessageEdited(id) {
-			this.messagesGroupedByAuthor = this.prepareMessagesGroups(this.messagesList)
+		handleMessageEdited(message) {
+			// soft edit for a message in the list
+			// find the corresponding date group id (dateTimeStamp)
+			const dateGroupId = moment(message.timestamp * 1000).startOf('day').unix()
+			const groups = this.messagesGroupedByDateByAuthor[dateGroupId]
+			if (!groups) {
+				// Message was edited already and this is message loading phase
+				return
+			}
+			// find the corresponding messages group in the list
+			const group = Object.values(groups).find(group => group.id <= message.id && (group.nextMessageId >= message.id || group.nextMessageId === 0))
+
+			if (!group) {
+				// Messages were not loaded yet
+				return
+			}
```

Comment:
This was added because `handleMessageEdited` was triggered while group messages were not loaded yet. Perhaps it was called because system message of messageX was loaded while the messages itself were not loaded yet (in the past).

Reply:
yes, it's an addition
maybe, because we don't have softMessage update yet, this happens:
- editMessage response -> processMessage triggers update -> we split the initial group
- message polling response -> processMessage triggers update -> we going through it again. but groups are already splitted, and edited message is alone

Reply:
Fix is there: https://github.com/nextcloud/spreed/pull/11638
doesn't call `EventBus.$emit('message-edited')` anymore, but message still could be alone in the group, so patch is useful, I believe

Reply:
Yes, I added it